### ISSUE TITLE
Mobile page views need to be higher up on the screen

### ISSUE THREAD

**Issue by futuresoup** [smartercleanup/duwamish#192]
In many instances, page views are too low on the screen and the user isn't aware that they need to scroll down to read the full page and interact in a certain way.

For example, when first landing on Hey Duwamish from an iphone, it looks like this
![welcome_current_toolow](https://cloud.githubusercontent.com/assets/1459051/11378148/aa06c95e-929e-11e5-983b-72c6b7d672cb.png)

Instead, it should look like this:
![welcome_ideal](https://cloud.githubusercontent.com/assets/1459051/11378147/aa005830-929e-11e5-95a8-2cd10b8ea495.png)

This way, the user gets the full introduction and can see the "Start Mapping" button without having to scroll. The map doesn't need to be fully displayed until they make this first interaction.

Let's say the user begins exploring the map, and selects a report icon to view. It currently looks like this:
![report_ideal](https://cloud.githubusercontent.com/assets/1459051/11378145/a9fc98a8-929e-11e5-8467-cd8b6a3a7ba5.png)

Once again, it's too low, and the user might not notice there's more content to read and not be aware that they should scroll.

It should instead look like this:
![report_current_toolow](https://cloud.githubusercontent.com/assets/1459051/11378146/a9fd0a90-929e-11e5-97cf-d95fdf06b78e.png)

In this example, the icon is all the way at the top of the map, so it's still visible, but the rest of the screen is dedicated toward displaying as much of the report as possible.

---

The bottom line as far as the design concept is that the user should have the fullest view of the map only when it's time to explore the map (except for the Submit a Report button),  and the fullest view of the pages when it's time to explore those pages (except for a small part of the map that lets them know where they are)

**Comment by maxburto** [comment]
I commited a style change that sets the map element to a hight of 175px when the body element has the class "content-visible".

**Comment by LukeSwart** [comment]
Cool! I just deployed this to our dev site. Feel free to test it out...

**Comment by futuresoup** [comment]
I've noticed two things when testing here: http://mobiletest.me/iphone_5_emulator/?u=http://dev.heyduwamish.org/

1) When you hit the ``Start Mapping`` button on the welcome screen, it takes you to a view where the map window size is very limited. This doesn't seem to be the case if you hit the ``close`` button on the window, only when you hit the blue ``Start Mapping`` button (which most users will probably do when faced with the two options for closing the window)

2) When you add a report, the vertical layout of the place buttons obstructs the map when trying to select a location.

In order to properly address the second issue, I think we need to re-work the reporting workflow slightly.

Instead of selecting report type first, the user selects their location on the map first with the fullest map view possible and a minimal, but obvious, sized dialog that says ``"First select the location of your report by dragging the map and placing the red pin"``

Once the location has been landed, the user doesn't need the map anymore and we can reduce the map view and dedicate the space toward optimizing the quality of their report.

**Comment by LukeSwart** [comment]
I think the above comment by @futuresoup should be its own issue or added to our dynamic form issue #127 

Closing this issue, because I believe it is resolved.
